cmd/tether/tether_linux.go

```diff
@@ -176,7 +189,7 @@ func (t *osopsLinux) backchannel(ctx context.Context) (net.Conn, error) {
 	}
 
 	// HACK: currently RawConn dosn't implement timeout so throttle the spinning
-	ticker := time.NewTicker(1000 * time.Millisecond)
+	ticker := time.NewTicker(10 * time.Millisecond)
```

Comment:
1s was a bug.  The tether should retry more frequently than the PL connection side.  @hickeng spotted this issue and couldnt tell me why it was set to 1s.